.editorconfig

```diff
@@ -25,3 +25,13 @@ indent_style = space
 [*.{yml,yaml}]
 indent_style = space
 indent_size = 2
+
+# Prevent unexpected automatic indentation when crafting test-cases
+[/testlib/src/main/resources/**]
```

Comment:
@nedtwigg I (again) got issues when copy-pasting from Jupiter actual results to test resoures, as IntelliJ was automatically reformatting my clipboard content.